### DISCUSSION TITLE
How to use the OVOS Dashboard?

### DISCUSSION THREAD

**Discussion by bencrazy** [OpenVoiceOS/ovos-buildroot#129] [Q&A] [answered]
Is there something that I'm missing with using the Dashboard? It seems straightforward, just hit the address that's presented once Dashboard has been enabled in Developer Settings (including port). However, I get the following response regardless of which browser I use:

> This site can’t provide a secure connection192.168.0.197 sent an invalid response.
> Try running Windows Network Diagnostics.
> ERR_SSL_PROTOCOL_ERROR

I guess I'm not sure what to expect from the dashboard either. Ultimately, what I'm trying to do is find a way to easily copy-paste the long-lived token from Home Assistant into the configuration so that I can set up the skill. (If there's a way to do it via CL then that would also work, but I've been so far unable to find the config file for it).

Thanks in advance

**Comment by j1nx** [comment]
The dashboard is very much wirk and progress at the moment. Not yet much you can do with it.

There is just an open PR that includes oauth qrcode support for the home assistant skill.

that allows you to scan a qrcode on the screen with your mobile, logging in on it and all magic happened.

also still wip but will look into include it in the next image for testing purposes.

**Comment by JarbasAl** [comment, marked as answer]
for anyone reading this in the  present day, the dashboard idea was discarded and is no longer an option
